localstack/services/firehose/provider.py

```diff
@@ -191,6 +192,10 @@ def get_search_db_connection(endpoint: str, region_name: str):
 
 
 class FirehoseProvider(FirehoseApi):
+    def __init__(self) -> None:
+        super().__init__()
+        self.kinesis_listeners: dict[tuple, dict[str, KinesisProcessorThread]] = {}
```

Comment:
nit: we could define a class-level variable with the type hints, and also add a short description of what the tuple/str keys represent.

Reply:
Thanks for the comment @whummer. Since this attribute was in the store, I assumed we had to preserve multi-account support. The key of the outer dictionary is a tuple `account, region` and the value is a `dict` as `arn:thread`.

Reply:
Thanks for clarifying. What I mean was - we could pull the variable with type hints and description out of the initializer:
```
class FirehoseProvider(FirehoseApi):
    # maps account/region tuple to dict of (stream name)->(stream processor thread)
    kinesis_listeners: dict[tuple, dict[str, KinesisProcessorThread]] = {}

    def __init__(self) -> None:
        ...
```